vector-im/element-x-android
# Allow polls to be edited

## Type of change


## Content

Allow polls to be edited if they have no votes.

## Motivation and context

- Part of https://github.com/vector-im/element-meta/issues/2179

## Screenshots / GIFs



## Tests

- Create a poll
- Edit the poll from the timeline
- Edit the poll from the message actions menu
- Vote on the poll and observe that it's no longer editable

## Tested devices

- OS version(s): 13

## Checklist
